tests/libraries/token/test_standard_token.py

```diff
@@ -36,4 +36,86 @@ def test_eternal_approval_magic(contractsFixture):
     assert standardToken.allowance(tester.a0, tester.a1) == 2 ** 256 - 1
     assert standardToken.balanceOf(tester.a0) == 4
     assert standardToken.balanceOf(tester.a1) == 96
-    
\ No newline at end of file
+    
+def test_create_negative_balance(contractsFixture):
+    standardToken = contractsFixture.uploadAndAddToController("solidity_test_helpers/StandardTokenHelper.sol")
+    assert standardToken.totalSupply() == 0
+    # We get some tokens for tester 0
+    assert standardToken.faucet(100)
+    assert standardToken.balanceOf(tester.a0) == 100
+    # Tester 1 tries to send tokens from tester 0 to himself and fails
+    with raises(TransactionFailed):
+        standardToken.transferFrom(tester.a0, tester.a1, 101)
+
+def test_transfer_then_create_negative_balance(contractsFixture):
+    standardToken = contractsFixture.uploadAndAddToController("solidity_test_helpers/StandardTokenHelper.sol")
```

Comment:
Uploading contracts can be a marginally time consuming operation.  Since you have several tests that all use this, consider creating a fixture with a snapshot like you did previously so you can just reset back to that state rather than having to re-upload each test.

Reply:
add test fixture to minimize the thrashing of uploading contract